src/screens/FeedScreen/Header.js

```diff
@@ -435,7 +532,22 @@ const styles = StyleSheet.create({
   avatarImage: {height: 48, width: 48, borderRadius: 24},
   postDetail: (isPostDetail) => ({
     paddingLeft: isPostDetail ? 10 : 0
-  })
+  }),
+  contentFeedName: {
+    flexDirection: 'row'
+  },
+  textFollow: {
+    color: colors.bluePrimary,
+    fontSize: normalizeFontSize(14),
+    fontStyle: 'normal',
+    fontWeight: '500'
+  },
+  textFollowing: {
+    color: colors.greySubtile1,
+    fontSize: normalizeFontSize(14),
+    fontStyle: 'normal',
+    fontWeight: '500'
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New styles have been added for the follow/unfollow button and the username container. These changes should be reviewed to ensure they align with the overall design and layout of the application.
```diff
+   contentFeedName: {
+     flexDirection: 'row'
+   },
+   textFollow: {
+     color: colors.bluePrimary,
+     fontSize: normalizeFontSize(14),
+     fontStyle: 'normal',
+     fontWeight: '500'
+   },
+   textFollowing: {
+     color: colors.greySubtile1,
+     fontSize: normalizeFontSize(14),
+     fontStyle: 'normal',
+     fontWeight: '500'
+   }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->